### PR TITLE
Added Windows Task Scheduler solution.

### PR DESCRIPTION
**Pull request by Lockszmith**
This solution does not require 3rd Party tools.

### CODE REVIEW

File: users/autostart.rst [hunk 1]
```diff
@@ -10,6 +10,32 @@ Windows
...
+    #. (optional) choose a delay
+    #. Make sure Enabled is checked
+    #. Click [OK]
+#. Actions Tab:
+    #. Click [New...]
+    #. [Action] should be set as [Start a program]
+    #. Enter the path to syncthing.exe in [Program/Script]
+    #. Click [OK]
+#. Settings Tab:
+    #. (recommended) Kepp the checkbox on [Allow task to be run on demand]
```
Comment (AudriusButkevicius) [inline users/autostart.rst]: Typo

File: users/autostart.rst [hunk 2]
```diff
@@ -10,6 +10,32 @@ Windows
 There is currently no official installer available for Windows. However,
 there are a number of easy solutions.
 
+Task Scheduler
+~~~~~~~~~~~~~~
+
+#. Start the `Task Scheduler <https://en.wikipedia.org/wiki/Windows_Task_Scheduler>`__
+#. Create a New Task ([Action] menu -> [Create Task...])
+#. General Tab:
+    #. Name the task (for example 'SyncThing')
```
Comment (AudriusButkevicius) [inline users/autostart.rst]: Its Syncthing not SyncThing

File: users/autostart.rst [hunk 3]
```diff
@@ -10,6 +10,32 @@ Windows
 There is currently no official installer available for Windows. However,
 there are a number of easy solutions.
 
+Task Scheduler
+~~~~~~~~~~~~~~
+
+#. Start the `Task Scheduler <https://en.wikipedia.org/wiki/Windows_Task_Scheduler>`__
+#. Create a New Task ([Action] menu -> [Create Task...])
```
Comment (AudriusButkevicius) [inline users/autostart.rst]: Suggest you use backticks rather than [ ] as it changes the formatting.

**Comment by calmh** [comment]
:+1:

**Pull request merged**